andela/mifune-cfh
# 145190873 UI Cleanup play buttons

#### What does this PR do?
Saves played game and retrieves saved games when needed
#### Description of Task to be completed?
UI Cleanup play buttons
#### How should this be manually tested?
Start a new game, play it to the end, once the game ends, you see a modal with game info if the game has been successfully saved.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
UI Cleanup play buttons
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
